### ISSUE TITLE
Multiple scipt entries in ui

### ISSUE THREAD

**Issue by oceanpuzzle** [godotengine/godot#41958]
After adding a script and selecting it I noticed in the lower right corner under the pause the section there was multiple script drop down controls. Restarting the engine it fixed itself but this seemed like bug. 

**Godot version:**
3.2.2

**OS/device including version:**

**Issue description:**

**Steps to reproduce:**
Not sure. After restart I was able to add a script and it was fine.

**Minimal reproduction project:**

**Comment by Calinou** [comment]
Can you take a screenshot when it happens?

**Comment by Calinou** [comment]
Reading OP again, I suppose they're referring to this section of the Inspector:

![Pause and Script sections in the editor inspector](https://user-images.githubusercontent.com/180032/94869705-27fc4c80-0446-11eb-8b26-80b18927a85d.png)

Has anyone else been able to reproduce this on Godot 3.2.3 or later?

**Comment by oceanpuzzle** [comment]
Correct. I had multiple controls under that section. I was following this
tutorial on youtube.
https://www.youtube.com/watch?v=Y_2oiLjOx54&t=1098s

Happened after adding the second script.
